src/main/java/org/folio/util/ErrorUtils.java

```diff
@@ -37,6 +38,9 @@ public static Error buildError(int status, ErrorType type, String message) {
         status = ((HttpException) cause).getCode();
         JsonObject errorJsonObject = ((HttpException) cause).getError();
         error = resolveError(errorJsonObject);
+      } else if (cause instanceof IllegalArgumentException) {
```

Comment:
Is it make sense to introduce custom exception, for example, ConverterException, for all thrown exceptions from Converters?

Reply:
introduced MarcConversionException